src/main/java/com/minelittlepony/hdskins/client/PlayerSkins.java

```diff
@@ -60,8 +61,8 @@ private void onCustomTextureLoaded(SkinType type, Identifier location, Minecraft
     }
 
     private void onVanillaTextureLoaded(Type type, Identifier location, MinecraftProfileTexture profileTexture) {
-        playerInfo.getVanillaTextures().put(type, location);
-        vanillaProfiles.put(SkinType.forVanilla(type), profileTexture);
+        vanillaTextures.put(SkinType.of(type.name()), location);
```

Comment:
vanillaTextures should be stored in the original map (with the original enum) to preserve compatibility with other mods and Mojang's code.

Reply:
It's easier to get here. Textures are still going there anyway.